Makefile

```
@@ -255,6 +255,10 @@ test-cov: $(PROMTOOL)
 test-cov-clean:
 	@./hack/test-cover-clean.sh
 
+.PHONY: check-api-diff
+check-api-diff: $(GO_APIDIFF)
+	@./hack/check-api-diff.sh
```

Let's call this `apidiff` just like the tool binary
```suggestion
.PHONY: check-apidiff
check-api-diff: $(GO_APIDIFF)
	@./hack/check-apidiff.sh
```